server/users.ts

```diff
@@ -579,7 +579,7 @@ export class User extends Chat.MessageContext {
 	popup(message: string) {
 		this.send(`|popup|` + message.replace(/\n/g, '||'));
 	}
-	getIdentity(roomid: string = '') {
+	getIdentity(roomid: RoomID = '' as RoomID) {
```

Comment:
```suggestion
	getIdentity(roomid = '' as RoomID) {
```

If you ever find yourself typing a type twice, you're probably underestimating TypeScript's ability to infer.